### ISSUE TITLE
[CoE Starter Kit - QUESTION] Power Platform Administration Planning Dashboard Documentation 

### ISSUE THREAD

**Issue by medicineshow** [microsoft/coe-starter-kit#3612]
### What is your question?

Can you please provide a link or detail regarding the Power Platform Administration Planning Dashboard that was shipped with the September release. Thank you.

### What solution version are you using?

September 2022

**Comment by Jenefer-Monroe** [comment]
Here is what we have written up. 
https://docs.microsoft.com/power-platform/guidance/coe/power-bi-compliance#compliance-and-governance

Thanks for using CoE

**Comment by medicineshow** [comment]
Thank @Jenefer-Monroe . I am slightly confused though. the documentation says install a solution first (which I only just saw):
https://docs.microsoft.com/en-us/power-platform/guidance/coe/setup-admin-tasks-component. 
I do not see the solution in the starter kit download.
![image](https://user-images.githubusercontent.com/17124284/189137159-420496eb-7704-4a08-babe-05e23ed04c9b.png)

Then there is the pbit file which I am unable to install and I assume that is because there is no solution installed?

**Comment by Jenefer-Monroe** [comment]
oh apologies. I thought you were asking about a different feature.

**Comment by manuelap-msft** [comment]
Hello,

that's curious - what link did you use to download the CoEStarterKit.zip file? I've just downloaded it now and it contains all the relevant files for the Admin Task Planning tool (highlighted below) so we may have an outdated link floating around somewhere that doesn't point to the right package. 

![image](https://user-images.githubusercontent.com/55705669/189142782-2561dad2-c463-49bd-9ad8-507fe55d52b5.png)

You can download the correct package that contains all the files from here https://github.com/microsoft/coe-starter-kit/releases/download/CoEStarterKit-September2022/CoEStarterKit.zip and if you let me know what link give you the incomplete .zip file I'll make sure to get that updated!

**Comment by medicineshow** [comment]
I actually got it from here:
https://github.com/microsoft/coe-starter-kit/releases/
and downloaded the CoEStarterKit.zip. But when I downloaded the same just now, it contains the file. I guess I was
 just too quick off the mark. I notice on the page it says 17 hours ago and I did download it about 24 hours ago.
![image](https://user-images.githubusercontent.com/17124284/189146540-6d13ff94-30cb-4470-b590-d22b1bf65c0e.png)
